cluster-customizer/initialize

```diff
@@ -166,12 +222,13 @@ setup
 require handler
 require files
 require network
-handler_add_libdir share
 require customize
+require customize-repository
+require ruby
 
 files_load_config cluster-customizer
 cw_CLUSTER_CUSTOMIZER_path="${cw_CLUSTER_CUSTOMIZER_path:-${cw_ROOT}/var/lib/customizer}"
-cw_CLUSTER_CUSTOMIZER_profiles="${cw_CLUSTER_CUSTOMIZER_profiles:-default}"
+cw_CLUSTER_CUSTOMIZER_account_profiles="${cw_CLUSTER_CUSTOMIZER_account_profiles:-default}"
```

Comment:
Could we add a little backwards compatibility in here and do something like this please?

```
${cw_CLUSTER_CUSTOMIZER_account_profiles:-${cw_CLUSTER_CUSTOMIZER_profiles:-default}}
```